src/main/commands/commandManager.ts

```diff
@@ -188,9 +188,10 @@ export class CommandManager implements ExtensionComponent {
 
     /**
      * Connect to JFrog Platform server. If the connection success, perform a quick scan.
+     * @param chooseMethod if true will open a quick pick to choose the connection method, false will try all the options
```

Comment:
if true will open a quick pick to choose the connection method, false will try all the options
-->
if true, a quick pick UI is shown, to choose the connection method. If false, all the connection methods are attempted.